### DISCUSSION TITLE
part-ii/2020-3/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#93] [Question comments]
Part II 2020 Algebraic Geometry: Paper 4, Section II, F 

https://questions.tripos.org/part-ii/2020-3/

**Comment by Kiolt** [comment]
<details>
<summary>First part.</summary>

Map the first n points to coordinates via projective equivalence, let the other two points be $\lambda,\mu$. Define $G(Z_0,Z_1)=\prod_{i=0}^n (\lambda_i^{-1}Z_1-\mu_i^{-1}Z_0)$, then the rational normal curve is $[Z_0,Z_1]\mapsto \left[ \frac{G}{\lambda_0^{-1}Z_1-\mu_0^{-1}Z_0},\ldots,\frac{G}{\lambda_n^{-1}Z_1-\mu_n^{-1}Z_0} \right]$.

</details>

<details>
<summary>Second part.</summary>

Take the basis $X_0^3,X_0^2X_1,X_0X_1^2,X_1^3$, the generators are $Z_0Z_3-Z_1Z_2, Z_0Z_2-Z_1^2,Z_1Z_3-Z_2^2$. To show that these generate use the relations to reduce any homogeneous poly to a linear combination of monomials of the types $Z_0^i Z_1^j$ and $Z_2^kZ_3^l$ and show that all the coefficients vanish.

</details>
